specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -4861,6 +4909,29 @@
         }
       }
     },
+    "RebillDetails": {
+      "description": "The rebill details of an invoice.",
+      "properties": {
+        "creditNoteDocumentId": {
+          "description": "The ID of credit note.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceDocumentId": {
+          "description": "The ID of invoice.",
+          "type": "string",
+          "readOnly": true
+        },
+        "rebillDetails": {
+          "description": "Rebill details for an invoice.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "$ref": "#/definitions/RebillDetails"
+          }
+        }
```

Comment:
is this field "readonly"? If it's, please add it

Reply:
Sure, I have added it. Thanks!